scripts/globals/mobskills/mix_guard_drink.lua

```diff
@@ -8,6 +8,9 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    target:addStatusEffect(xi.effect.PROTECT, 220, 0, 300)
+    target:addStatusEffect(xi.effect.SHELL, 2930, 0, 300)
```

Comment:
https://ffxiclopedia.fandom.com/wiki/Trust:_Monberaux
https://www.bg-wiki.com/ffxi/Monberaux

The wikis agree on 220 for the Protect value, but where did you get the Shell values for this from? Did you get this from retail? Protect 220 vs Shell 2930 doesn't seem right (though, I don't really know much about the values of these sorts of thing)

Does this scale with level?

Reply:
@zach2good wiki's show a x/256ths scale for shell where we use a larger basis to do a percentage down to multiple decimal points, hence the discrepancy. So what he has here is 29.30%

Reply:
Teo is right, 2930 for Shell is 29.30%. I referenced the values for Protect and Shell V, which BGWiki says is the potency of the guard drink move.